magma/digital.py

```diff
@@ -154,7 +161,7 @@ def wire(self, o, debug_info):
         if isinstance(o, IntegerTypes):
             o = HIGH if o else LOW
 
-        if not isinstance(o, Digital):
+        if not isinstance(magma_value(o), Digital):
```

Comment:
same here as above, should we just do `o = magma_value(o)` so `o._wire` is valid?

Reply:
deleted with `o = magma_value(o)`